CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ This change log adheres to [keepachangelog.com](http://keepachangelog.com).
 * moved all logging and CLI-related logic out of API and into CLI
 * **breaking:** changed API parameters from `options, auth, callback` to `options, callback`
   * `auth` is now in `options` as `options.auth`
+* no longer uses the node-github client
+### Added
```

Comment:
@paulcpederson You need a newline between lists and headers in markdown otherwise it looks like this:

https://github.com/paulcpederson/gh-release/blob/assets/CHANGELOG.md#changed